api/scpca_portal/test/management/commands/test_load_data.py

```diff
@@ -192,8 +201,8 @@ def test_merged_project_anndata_cite_seq(self):
             # └── single_cell_metadata.tsv
             files = set(project_zip.namelist())
             self.assertEqual(len(files), 8)
-            self.assertIn("SCPCP999992_merged.rds", files)
-            self.assertNotIn("SCPCP999992_merged_adt.h5ad", files)
+            self.assertIn("merged/SCPCP999992_merged.rds", files)
```

Comment:
Update the contents comment just above this line

Reply:
Good catch. Upon referring to documentation, it looks like merged data files should be nested under the zip root directory, not under a nested `merged/` directory inside the zip root. I've made the change accordingly to transform merges paths to reflect this.